### PR TITLE
allow building valhalla without services

### PR DESCRIPTION
**Pull request by rinigus**
Addresses https://github.com/valhalla/valhalla/issues/706

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -48,6 +48,21 @@ AX_BOOST_IOSTREAMS
...
+    [case "${enableval}" in
+        yes) services=true ;;
+        no)  services=false ;;
+        *) AC_MSG_ERROR([bad value ${enableval} for --enable-services]) ;;
+    esac],[services=true])
+AM_CONDITIONAL([SERVICES], [test x$services = xtrue])
+
+# then we require prime_server
+AM_COND_IF([SERVICES], [
+    PKG_CHECK_MODULES([DEPS], [protobuf >= 2.4.0 libprime_server = 0.6.3 libcurl >= 7.35.0])
```
Comment (kevinkreiser) [inline configure.ac]: protobuf is required regardless, so you need to move just protobuf into a check outside of this condition

Comment (kevinkreiser) [inline configure.ac]: so maybe you want to call this one SERVICE_DEPS and then just put the protobuf one into DEPS outside of this condition

**Comment by kevinkreiser** [comment]
other than comments this looks good

**Comment by rinigus** [comment]
My mistake - thank you for catching it. I'll look into it later tonight.

**Comment by rinigus** [comment]
Hi! It should be ready for review now.

**Pull request merged**